# Mission Config of the sub
# ALL UNITS IN M

# Search Algorithms: simple

tasks:
 start_gate:
  prior: [8,-1,-1.5]
  search_alg: simple
  dist_behind_gate: 1.0
  failed_repeat: true
  object_localizers:
    start_gate_pole: ['gate_pnp_color', 'gate_pnp_bb', 'gate_midpoint']
 dice:
  prior: [3,0,-2.3]
  search_alg: simple
  failed_repeat: false
  targets: ['dice5', 'dice6']
  replan_threshold: 0.5
  approach_method: 'direct' # optimal, twopoint, direct
  approach_radius: 1.5
  z_mode: 'pose' # pose, nudging, bangbang
  orbital_radius: 2.0
  lockon_time: 5.0
  carrot_dist: 1.5
  backup_dist: 5.0
  accel_spike_thresh: -0.15
  accel_axis: 'x'
  attack_timeout: 15
  object_localizers:
    dice1: dice_box
    dice2: dice_box
    dice5: dice_box
    dice6: dice_box

mission_tasks: # Tasks To Do In Order
 - dice
